src/main/java/com/example/crawlertest/repositories/VacatureRepository.java

```diff
@@ -1,15 +1,21 @@
 package com.example.crawlertest.repositories;
 
 import com.example.crawlertest.domein.Vacature;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
 import java.util.Optional;
 
 @Repository
-public interface VacatureRepository extends CrudRepository<Vacature, Long> {
+public interface VacatureRepository extends PagingAndSortingRepository<Vacature, Integer> {
```

Comment:
Kun je vacatures nog opslaan met deze nieuwe Repository? Denk dat we beter JpaRepository kunnen gebruiken: https://www.baeldung.com/spring-data-repositories